setup.py

```diff
@@ -24,29 +24,16 @@
 # WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
-import ast
-import io
 import os
 import re
+import codecs
 from setuptools import setup, find_packages
 
 
-class VersionFinder(ast.NodeVisitor):
-
-    def __init__(self):
-        self.version = None
-
-    def visit_Assign(self, node):
-        if node.targets[0].id == '__version__':
-            self.version = node.value.s
-
-
 def read_version():
     """Read version from httpretty/version.py without loading any files"""
-    finder = VersionFinder()
-    finder.visit(
-        ast.parse(local_file('httpretty', '__init__.py').encode('utf-8')))
-    return finder.version
+    contents = local_file('httpretty', '__init__.py')
+    return re.compile(r".*__version__ = version = '(.*?)'", re.S).match(contents).group(1)
```

Comment:
this is a lot less reliable than using the python's abstract syntax tree, can we keep it the old way ? :)